<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find your location</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="countryLabel">
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="countryText"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City</string>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cityText"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="regionLabel">
       <property name="text">
        <string>Region</string>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="regionText"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="latLabel">
       <property name="text">
        <string>Lat</string>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="latText"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lonLabel">
       <property name="text">
        <string>Lon</string>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lonText"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ipLabel">
       <property name="text">
        <string>Public IP</string>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="ipText"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
